### DISCUSSION TITLE
Observables explained

### DISCUSSION THREAD

**Discussion by neu5** [Annoraaq/grid-engine#65] [General]
How about some examples with using observables from the API and more detailed documentation of them?

**Comment by Annoraaq** [comment]
Hi @neu5,

there are examples for all of the observables:

movementStarted, movementStopped, directionChanged: https://annoraaq.github.io/phaser-grid-movement-plugin/examples/custom-walking-animation.html

positionChanged: https://annoraaq.github.io/phaser-grid-movement-plugin/examples/position-changed.html

Can you give a hint about what exactly is unclear about the observables in the docs? That helps us completing it.

**Reply by Annoraaq** [reply]
Also, I think @splashsky is currently working on a more detailed documentation. It might be, that there will me more information about the observables.

**Reply by neu5** [reply]
Sorry for bothering you! I must have been looking at old version of the examples and I didn't see that before. That's all what I would expect from documentation

**Reply by Annoraaq** [reply]
No need to apologize! It's a totally valid suggestion. And I admit that it is also not necessarily self-explanatory which example contains what exactly. But the docs will get more attention in the future.

**Reply by splashsky** [reply]
I'll work on adding some comments to the upcoming documentation, so the code becomes a little more obvious. 😄

**Reply by borkowsky** [reply]
Links not work(

**Reply by Annoraaq** [reply]
Sorry we moved the examples. I just updated the links here.
